### DISCUSSION TITLE
The Pages API has been broken for over a month.

### DISCUSSION THREAD

**Discussion by easrng** [community/community#12768] [Other Feature Feedback, Questions, & Ideas]
I'm copying this [from the support community](https://github.community/t/pages-api-seems-to-be-broken/224202?u=easrng) in case this is monitored more by GH staff.
The Pages API has been broken for over a month. Requesting any `https://api.github.com/repos/:user/:repo/pages` URL, **including the [example url from the docs](https://api.github.com/repos/octocat/hello-world/pages)** return this:
```json
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest/reference/repos#get-a-github-pages-site"
}
```

What gives?

**Comment by jfullstackdev** [comment]
seems this is still not fixed, I have a GitHub Pages enabled at this repo yet it's 404 

![image](https://user-images.githubusercontent.com/47092464/182762929-b50dec6b-f2eb-4420-8d08-add2c8247f1b.png)
